spec/problem_one_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../problem_one'
+
+describe ProblemOne do
+  it 'computes for a set of even numbers' do
+    even_set = [2, 8, 12, 10, 128, 64, 32]
+    even_sum = subject.sum_even(even_set)
+    expect(even_sum).to eq(256)
+  end
+
+  it 'computes for a set of odd numbers' do
+    odd_set = [1, 5, 11, 3, 1337, 9]
+    odd_sum = subject.sum_even(odd_set)
```

Comment:
Why is this an add sum? shouldn't it be the sum of even numbers?

Reply:
It is. The test was meant to handle the case where the result is `0`, as the set contains no even numbers. The test failed in https://github.com/Agilefreaks/Test/pull/42/commits/8adf20673b7f88057108882b0fe11a48576d0650 (RED), which led to handling `0` as a case https://github.com/Agilefreaks/Test/pull/42/commits/ba0cc510bd226148c97dcece41722c8ae97f8545 (GREEN).

`empty_sum` was added afterwards, but it did not undergo the RED-stage, meaning that was a TDD error on my behalf. Will remove `empty_sum` as it is irrelevant and also improve the naming.